### DISCUSSION TITLE
整理视频时无法处理10以上的集数

### DISCUSSION THREAD

**Discussion by CRAZYShimakaze** [jxxghp/MoviePilot#2645] [问题咨询]
如图所示,09集可正确识别,但10集会连同集数一起当做标题进行处理.

![image](https://github.com/user-attachments/assets/cf85e3ae-ddf3-4b19-a7d2-2b17d0f26916)

**Comment by MoogleXX** [comment]
这种是命名不标准的问题。自定义识别词吧。
初步看你的这个。不知道你这个剧集是多少季的。假设是第一季吧。“[ANK-Raws]ノーゲーム・ノーライフ => [ANK-Raws]ノーゲーム・ノーライフ S01E”这样就相当于识别时候把你这一集名字强制定义为“[ANK-Raws]ノーゲーム・ノーライフ S01E10”去了。
另外[]符号好像需要特殊字符转义。可能需要写成\[ 和 \]。实际你试试看吧。在MP右上角识别里面的识别名称测试就知道了。
![11](https://github.com/user-attachments/assets/e1bde259-ef77-4497-be43-f4d902c7aa53)
